unyt/tests/test_array_functions.py

```diff
@@ -249,6 +238,12 @@ def get_wrapped_functions(*modules):
     return dict(sorted(wrapped_functions.items()))
 
 
+def assert_array_equal_units(a, b):
```

Comment:
Why can't you use the version of this that's already in `test_unyt_array`? Totally fine to modify that one to work with ndarrays too.

Reply:
mostly because I expected it to live in a public API module and wanted to avoid importing from a test module.
Can I move the definition to `unyt/testing.py` ?

Reply:
please do